rpg.py

```diff
@@ -57,16 +64,24 @@ def act(self, players: List) -> List:
             players[targeted_index] = damaged_player
         return players
 
+    def view_stats(self, players: List) -> List['Character']:
+        for player in self.players:
+            if character.is_alive() and character.team == 1:
+                print(f'{player.name} LV: {player.level}')
+                print(f'HP: {player.hp}/{player.max_hp}')
+                print(f'Attack: {player.attack}')
+                print(f'Speed: {player.speed}')
+            return players
```

Comment:
You are returning inside the loop.

That means you will only look at the first character in the list before ending.

Also `self.players` isn't an attribute of `Character`